keps/sig-node/20190806-memory-manager.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Memory Manager
+authors:
+  - "@bg-chun"
+  - "@ohsewon"
+  - "@admanV"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+reviewers:
+  - TBD
+  - TBD
+approvers:
+  - TBD
+  - TBD
+editor: TBD
+creation-date: 2019-08-06
+last-updated: 2019-08-06
+status: draft
+see-also:
+replaces:
+superseded-by:
+---
+
+# Memory Manager
+
+_Authors:_
+
+- @bg-chun - Byonggon Chun &lt;bg.chun@samsung.com&gt;
+- @ohsewon - Sewon Oh &lt;sewon.oh@samsung.com&gt;
+- @admanV - Hyunsung Cho &lt;hs2.cho@samsung.com &gt;
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Motivation](#motivation)
+  - [Related Features](#Related-features)
+  - [Related issues](#Related-issues)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+  - [User Stories](#user-stories)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+- [Proposal](#proposal)
+  - [Proposed Changes](#proposed-changes)
+    - [New Component: Memory Manager](#new-component-memory-manager)
+      - [Calculate NUMA node affinity for memory and hugepages](#calculate-numa-node-affinity-for-memory-and-hugepages)
+      - [Provide topology hint for Topology Manager](#provide-topology-hint-for-topology-manager)
+      - [Isolate memory and hugepages for a container](#isolate-memory-and-hugepages-for-a-container)
+      - [New Interfaces](#new-interfaces)
+    - [Topology Manager changes](#topology-manager-changes)
+    - [Internal Container Lifecycle changes](#internal-container-lifecycle-changes)
+    - [Feature Gate and Kubelet Flags](#feature-gate-and-kubelet-flags)
+- [Graduation Criteria](#graduation-criteria)
+  - [Phase 1: Alpha (target v1.1x)](#phase-1-Alpha-target-v11x)
+  - [Phase 2: Beta](#phase-1-Beta)
+  - [GA (stable)](#ga-stable)
+
+# Overview
+
+NUMA Awareness is well known for a solution to boost performance for diverse use cases including DPDK and Database. Kubernetes support NUMA related features for NUMA sensitive containers.
```

Comment:
I think NUMA is better because Topology Manager, CPU Manager and, device plugin they are calculating topology hint based on NUMA node where resources exist.